pynest/nest/lib/hl_api_helper.py

```diff
@@ -69,7 +69,8 @@
 # deprecated functions will be registered by the @deprecated decorator.
 _deprecation_warning = {'BackwardCompatibilityConnect': True,
                         'subnet': True,
-                        'aeif_cond_alpha_RK5': True}
+                        'aeif_cond_alpha_RK5': True,
+                        'iaf_psc_alpha_canon': True}
```

Comment:
Furthermore, deprecate `iaf_psc_alpha_presc`. This model will not be replace with anything else, it was just a temporary test model for the Morrison et al 2007 paper.